springwolf-asyncapi/src/main/java/io/github/stavshamir/springwolf/asyncapi/v3/model/channel/message/MessageObject.java

```diff
@@ -118,4 +118,14 @@ public class MessageObject extends ExtendableObject implements Message {
      */
     @JsonProperty(value = "traits")
     private List<MessageTrait> traits;
+
+    /*
+     * Override the getMessageId to guarantee that there's always a value. Defaults to 'name'
+     */
+    public String getMessageId() {
```

Comment:
The `messageId` is **not** part of the `MessageObject` definition. The AsyncAPI v3 defines `messageId` as a key when a `Message` is part of a `Map`.

The `messageId` field was added to automatically build the maps when needed, but this method was needed to fix some cases.

Should we take a similar approach as the one with `ChannelId` and also remove this value and _always_ use the `name` field?